### PR TITLE
refactored sandbox page to overmind

### PR DESCRIPTION
**Pull request by christianalfoni**
Tested opening up the page, though think @CompuIves should check live and routing behaviour here, as I am a bit unsure

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3180/commits/8133a9ec98bf0ac0eaab628bd4088046fc478a8b is at https://pr3180.build.csb.dev/s/new.

File: packages/app/src/app/pages/Sandbox/index.tsx [hunk 1]
```diff
@@ -3,21 +3,45 @@ import Centered from '@codesandbox/common/lib/components/flex/Centered';
...
 import { Title } from 'app/components/Title';
+import { useOvermind } from 'app/overmind';
 import { GithubIntegration } from 'app/pages/common/GithubIntegration';
 import { Navigation } from 'app/pages/common/Navigation';
 import { NotFound } from 'app/pages/common/NotFound';
 import { QuickActions } from 'app/pages/Sandbox/QuickActions';
-import React from 'react';
-import Helmet from 'react-helmet';
+import React, { useCallback, useEffect } from 'react';
+import { Helmet } from 'react-helmet';
```
Comment (CompuIves) [inline packages/app/src/app/pages/Sandbox/index.tsx]: Whoh, this doesn't break anything?

Comment (christianalfoni) [reply packages/app/src/app/pages/Sandbox/index.tsx]: Nopes 😄

**Comment by CompuIves** [comment]
Works great!!!

**Pull request merged**